src/function.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-loop-func */
+let addlists = [];
+let isEditing = false;
+let todoEdit = null;
+const save = () => {
+  localStorage.setItem('addlists', JSON.stringify(addlists));
+};
+
+const getData = () => {
+  const listData = JSON.parse(localStorage.getItem('addlists'));
+  if (listData == null) {
+    addlists = [];
+  } else {
+    addlists = listData;
+  }
+};
+const toggleListCheck = (todo) => {
+  addlists = addlists.map((addlist_) => {
+    if (addlist_.index === todo.index) {
+      return { ...todo, completed: !addlist_.completed };
+    }
+    return addlist_;
+  });
+  save();
+};
+
+const displayList = () => {
+  const listElement = document.getElementById('lists-todo');
+  listElement.innerHTML = '';
+  for (let i = 0; i < addlists.length; i += 1) {
```

Comment:
- [ ] [OPTIONAL] In accordance with the KISS rule, you could use the `forEach()` method here to keep the code simple.